src/anders_mathiesen_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+from random import randint
+
+__author__ = 'Anders Mathiesen'
+__email__ = 'andermat@nmbu.no'
+
+
+def get_guess_bigger_then_or_lik_1():
+    guess = 0
+    while guess < 1:
+        guess = input('Your guess bigger then or lik 1: ')
+        try:
+            guess = int(guess)
+        except:
```

Comment:
This is pretty cool, though be wary of using a "bare" except statement as it is usually supposed to catch errors. A better option in this case would be an if-test.